guidance/guidance_los/scripts/los_guidance_action_server.py

```diff
@@ -86,6 +54,8 @@ def __init__(self):
         self._declare_topic_parameters()
         self._initialize_publishers()
         self._initialize_subscribers()
+        self._initialize_state()
+        self._initialize_action_server()
 
         # Initialize guidance calculator with third-order filtering
         self.guidance_calculator = ThirdOrderLOSGuidance(self.los_params)
```

Comment:
Remember to actually declare and get the parameters to create `self.los_params` ;)